### PR TITLE
Add the Cully Jazz festival

### PR DESCRIPTION
**Pull request by MarkJaroski**
A Jazz festival on Lac Léman which is really Jazz as opposed to Montreux.

This will add some big names and some local acts to the feed.

### CODE REVIEW

File: croncert-config.yml [hunk 1]
```diff
@@ -412,6 +412,53 @@ scrapers:
...
+  ############
+  # Cully
+  ############
+  - name: CullyJazz
+    url: https://cullyjazz.ch/fr/programme
+    item: body > div > main > div > div > div > section.row > div.event-box-wrapper > div.paid-events > div.event-box > div.event-box-inner > a
+    fields:
+      - name: city
+        value: Cully
+      - name: type
```
Comment (jakopako) [inline croncert-config.yml:424]: this has to be 'type' instead of 'evenType' otherwise the event api will reject this.

Comment (MarkJaroski) [reply croncert-config.yml:424]: OK. I think I'm confused about the difference between the two. Do you just use eventType for filters? And type is for the API?

Comment (jakopako) [reply croncert-config.yml:424]: type is a mandatory field if we want to write to the API which is the case for this repository, see https://github.com/jakopako/event-api/blob/main/models/Event.go#L22 

eventType is an arbitrary name that I sometimes use to filter events. I admit that the naming here is a bit suboptimal.

**Comment by MarkJaroski** [comment]
OK. I think I finally get it. I'll do it that way for the next ones.

**Comment by jakopako** [comment]
cool, no worries :)

**Pull request merged**